tests/test_container_provider.py

```diff
@@ -40,82 +52,100 @@ class AnsibleModuleFailed(Exception):
 
 @pytest.fixture()
 def miq(miq_api_class, miq_ansible_module, the_provider):
+
+    # TODO(jaryn) This would ideally be inside test cases that need it.
     miq_ansible_module.params = {'metrics': True}
 
     def fail(msg):
         raise AnsibleModuleFailed(msg)
 
     miq_ansible_module.fail_json = fail
-    miq = manageiq.ManageIQ(miq_ansible_module, "http://themanageiq.tld",
+    miq = manageiq.ManageIQ(miq_ansible_module, manageiq_hostname,
                             "The username", "The password")
 
-    miq_api_class.return_value.post.return_value = dict(results=[dict(id=0)])
+    miq_api_class.return_value.post.return_value = dict(results=[
```

Comment:
can we create another post return value for a delete action? (returns different fields)